client/src/component/admin/__test__/MenuListContainer.test.js

```diff
@@ -0,0 +1,35 @@
+// not working
+
+import { render, screen } from "@testing-library/react";
+
+import * as api from "../../../api/adminSideApi";
+import MenuListContainer from "../MenuListContainer";
+
+// // we want to run a function or some other code repeatedly “before each” test that code can be put in the beforeEach function.
+beforeEach(() => {
+  //jest.spyOn: Spy or mock a function
+  jest.spyOn(api, "getItemList").mockResolvedValue([]);
+});
+
+describe("render", () => {
+  it("loads all the items", async () => {
+    api.getItemList.mockResolvedValue([
+      {
+        active: true,
+        categoryname: "Breakfast",
+        description: "7.30am to 6.30pm",
+        id: 6,
+        itemname: "Parota",
+        price: 60,
+      },
+    ]);
+
+    render(<MenuListContainer />);
+
+    // getting error  Expected number of calls: >= 1
```

Comment:
If you want `getItemList` to be called, you need to make sure your effect is triggered:

https://github.com/priyaraj7/guruprasad/blob/6fc6b9a8402ec6c22c526b3cc37e9c017875bbe7/client/src/component/admin/MenuListContainer.js#L32-L35

According to the above, there needs to be a `token` before the api is called.